### PR TITLE
[PB-748]: Feat/bump tailwind css

### PR DESCRIPTION
**Pull request by xabg2**
Update the TailwindCSS version to the latest one.

**SOME FUNCTIONALITIES OF THE GRID VIEW HAVE BEEN CORRECTED:**
   * Dropdown menú:
      * Open Share dialog
      * Preview files
      * Rename items
      * Copy link
      * Move item
      * Download item
      * Move to trash
   
  * 2 shortcuts:
     * Backspace -> delete selected item
     * R -> rename selected item
     
I've also updated some styles following the Tailwind update guide: https://tailwindcss.com/docs/upgrade-guide

### CODE REVIEW

**Review by aboredvaro — CHANGES_REQUESTED** [review]
Firstly, great work @masterprog-cmd 👏🏻👏🏻👏🏻

At first sight it looks like everything CSS related works, however the tailwindcss.config.json still needs to be upgraded, here's a guide on how to do it: https://tailwindcss.com/docs/upgrade-guide

On the other side, since grid view was improved, I propose to solve a bug too. The bug consist on the right click on grid view, it should open the same context menu as in list mode.

**Comment by xabg2** [comment]
> Firstly, great work @masterprog-cmd 👏🏻👏🏻👏🏻
> 
> At first sight it looks like everything CSS related works, however the tailwindcss.config.json still needs to be upgraded, here's a guide on how to do it: https://tailwindcss.com/docs/upgrade-guide
> 
> On the other side, since grid view was improved, I propose to solve a bug too. The bug consist on the right click on grid view, it should open the same context menu as in list mode.

I've fixed the bug @aboredvaro !

File: tailwind.config.js
Comment (aboredvaro) [inline tailwind.config.js:1]: Variants should be removed too: https://tailwindcss.com/docs/upgrade-guide#remove-variant-configuration

Please @masterprog-cmd complete the whole upgrade guide.

Screenshot 2023-11-02 at 12 35 32

File: src/app/shared/components/Empty/Empty.tsx [hunk 1]
```diff
@@ -22,8 +22,10 @@ export default function Empty({ icon, title, subtitle, action, contextMenuClick
       <button
         onClick={action.onClick}
         className={`mx-auto flex items-center rounded-lg ${
-          action.style === 'elevated' ? 'mt-5 bg-primary' : 'mt-2.5 bg-transparent'
-        } h-10 bg-opacity-10 px-5 font-medium text-primary hover:bg-opacity-15 active:bg-opacity-20`}
+          action.style === 'elevated'
+            ? 'mt-5 bg-primary/10 hover:bg-primary/15 active:bg-primary/20'
```
Comment (aboredvaro) [inline src/app/shared/components/Empty/Empty.tsx]: `hover:bg-primary/15 active:bg-primary/20` should be removed from this line and added back to the line 28

**Review by aboredvaro — CHANGES_REQUESTED** [review]

File: src/app/drive/components/DriveItemInfoMenu/DriveItemInfoMenu.tsx [hunk 2]
```diff
@@ -15,42 +15,42 @@ const DriveItemInfoMenu = (props: DriveItemInfoMenuProps): JSX.Element => {
...
-      <div className="flex items-center mb-4">
-        <div className="flex items-center min-w-9 w-9 h-9">
+      <div className="mb-4 flex items-center">
+        <div className="min-w-9 flex h-9 w-9 items-center">
           <props.icon className="h-full" />
         </div>
         <span
-          className="mx-3 overflow-hidden whitespace-nowrap\
-         overflow-ellipsis block text-neutral-700 text-sm flex-grow"
+          className="whitespace-nowrap\ mx-3 block
```
Comment (aboredvaro) [inline src/app/drive/components/DriveItemInfoMenu/DriveItemInfoMenu.tsx]: Remove the "\" at the end of `whitespace-nowrap\`

**Review by aboredvaro — CHANGES_REQUESTED** [review]

**Review by aboredvaro — APPROVED** [review]

**Pull request merged**